src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -141,7 +203,7 @@ const SidebarView = View.extend({
     this.showChildView('save', new DisabledSaveView());
   },
   showForm() {
-    if (this.model.isNew()) {
+    if (this.canEdit()) {
```

Comment:
`onSave` now re-renders which will call showForm again duplicating the listener on `this.model`.   
Also we need to be cloning this model for editing.. so if you make edits to the clinician name/email and cancel, it  doesn't modify the clinician model in memory.  Currently if you make an edit to the name and cancel, it will remain edited until refetched.
Similarly cancel currently just closes the sidebar, which works for new clinicians, but not for existing clinicians